### PR TITLE
feat: Create a new conversation from the contact panel

### PR DESCRIPTION
**Pull request by nithindavid**
Adds new conversation button on contact sidebar on contacts page

### CODE REVIEW

File: app/javascript/dashboard/routes/dashboard/contacts/components/ContactInfoPanel.vue [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <span class="close-button" @click="onClose">
       <i class="ion-android-close close-icon" />
     </span>
-    <contact-info :contact="contact" />
+    <contact-info :show-new-message="true" :contact="contact" />
```
Comment (pranavrajs) [inline app/javascript/dashboard/routes/dashboard/contacts/components/ContactInfoPanel.vue:6]: show-new-message is enough, I guess

File: app/javascript/dashboard/routes/dashboard/conversation/contact/ContactInfo.vue [hunk 2]
```diff
@@ -50,19 +50,41 @@
...
       >
         {{ $t('EDIT_CONTACT.BUTTON_LABEL') }}
       </woot-button>
+      <div v-else class="contact-actions">
+        <woot-button
+          class="new-message"
+          size="small expanded"
+          @click="toggleConversationModal"
+        >
+          {{ 'New conversation' }}
```
Comment (pranavrajs) [inline app/javascript/dashboard/routes/dashboard/conversation/contact/ContactInfo.vue:67]: i18n

File: app/javascript/dashboard/routes/dashboard/conversation/contact/ContactInfo.vue [hunk 3]
```diff
@@ -164,8 +196,22 @@ export default {
...
+  margin-right: var(--space-small);
+}
 
-.edit-contact {
-  margin-left: var(--space-slab);
+.contact-actions {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  /* padding-left: var(--space-medium); */
```
Comment (pranavrajs) [inline app/javascript/dashboard/routes/dashboard/conversation/contact/ContactInfo.vue:215]: 👃

File: app/javascript/dashboard/routes/dashboard/conversation/contact/ConversationForm.vue [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+    },
+    onSubmit: {
+      type: Function,
+      default: () => {},
+    },
+  },
+  data() {
+    return {
+      name: '',
+      phoneNumber: '',
```
Comment (pranavrajs) [inline app/javascript/dashboard/routes/dashboard/conversation/contact/ConversationForm.vue:99]: unused.

**Comment by muhsin-k** [comment]
@nithindavid button desgin is breaking in the smaller screens?
photo_2021-04-16 19 08 31

**Comment by muhsin-k** [comment]
@nithindavid Increase the height of input area
Screenshot 2021-04-16 at 7 17 14 PM

**Review by muhsin-k — APPROVED** [review]

**Comment by XuanThuLe** [comment]
Ok

**Pull request merged**